local_settings_example.py

```diff
@@ -1,2 +1,7 @@
-USER='myusername'
-PASSWORD='mypassword'
+# OpenFoodFacts username and password for authentication
+USER = 'myusername'
+PASSWORD = 'mypassword'
+# Change Postgres user, password and host as per requirement
+POSTGRES_USER = 'myPostgresUsername'
+POSTGRES_PASSWORD = '' # Leave empty if no password exists for user
+POSTGRES_HOST = 'myPostgresHost' # Usually "localhost"
```

Comment:
if it's usually localhost, put "localhost" directly :-) (sensible defaults)

Reply:
Yep done!